hibernate-c3p0/src/main/java/org/hibernate/c3p0/internal/C3P0ConnectionProvider.java

```diff
@@ -121,31 +125,32 @@ public void configure(Map<String, Object> props) {
 				JdbcSettings.URL,
 				JdbcSettings.JPA_JDBC_URL
 		);
+
 		final Properties connectionProps = ConnectionProviderInitiator.getConnectionProperties( props );
 
-		C3P0_MSG_LOGGER.c3p0UsingDriver( jdbcDriverClass, jdbcUrl );
-		C3P0_MSG_LOGGER.connectionProperties( ConfigurationHelper.maskOut( connectionProps, "password" ) );
+		ConnectionInfoLogger.INSTANCE.connectionProperties( ConfigurationHelper.maskOut( connectionProps, "password" ) );
 
 		autocommit = getBoolean( JdbcSettings.AUTOCOMMIT, props );
-		C3P0_MSG_LOGGER.autoCommitMode( autocommit );
 
 		if ( jdbcDriverClass == null ) {
-			C3P0_MSG_LOGGER.jdbcDriverNotSpecified();
+			ConnectionInfoLogger.INSTANCE.jdbcDriverNotSpecified();
 		}
 		else {
 			try {
 				serviceRegistry.requireService( ClassLoaderService.class ).classForName( jdbcDriverClass );
 			}
 			catch (ClassLoadingException e) {
-				throw new ClassLoadingException( C3P0_MSG_LOGGER.jdbcDriverNotFound( jdbcDriverClass ), e );
+				throw new ClassLoadingException( ConnectionInfoLogger.INSTANCE.jdbcDriverNotFoundErrMsg( jdbcDriverClass, e ) );
```

Comment:
We can just remove this log message. Logging and throwing at the same time is simply redundant.

Reply:
Well, I'm not logging *and* throwing, I'm only throwing, but with an additional message provided by the logger. I think we do this in many places thourhgout the code base?

Reply:
Excuse me, I misunderstood. 

But then, no, I don't think we do this elsewhere in the codebase, and I don't think we should do it here. There are thousands of places where we throw an exception, and none of them ever ask a logger to render the exception message.

Looks like it's only `C3P0MessageLogger` and `ProxoolMessageLogger` which ever use a `@Message` which is not a `@LogMessage`.

Reply:
Ok, no problem, I actually didn't change this wrt the original implementation, just moved it to the `ConnectionInfoLogger`, but i'll remove those.

Reply:
Yeah, yeah, I realize that.